.eslintrc.json

```diff
@@ -91,10 +91,14 @@
     "class-methods-use-this": "off",
     "no-return-await": "off",
     "no-warning-comments": "off",
-    "no-unused-vars": [
-      "error",
+    // note you must disable the base rule as it can report incorrect errors
```

Comment:
this change is to ignore parameters that start with `_` if they're unused, specifically for the case of the lambda handler that should have a context parameter in it's signature, but doesn't use it